### ISSUE TITLE
Similar Repos!

### ISSUE THREAD

**Issue by aquaflamingo** [cjgdev/smart-contract-patterns#1]
Hey there @cjgdev, i'm working on a similar repo [here](https://github.com/robertsimoes/Solidity-Contract-Patterns). 

Would you  mind if I import some of the patterns here?

**Comment by cjgdev** [comment]
Of course, I would be delighted to have my work contributed. It would be
nice to get a mention, but by all means please do as you please.
